### PR TITLE
ENHANCED CHROMATIC FORESIGHT

### PR DESCRIPTION
**Pull request by dequis**
ENHANCED CHROMATIC FORESIGHT

### CODE REVIEW

**Comment by Xe** [comment]
Critical to business.

File: templates/index.html [hunk 1]
```diff
@@ -32,6 +32,9 @@
                 display: block;
                 margin-bottom: 2em;
             }
+            #bg-color {
+                text-shadow: 1px 1px #AAAAAA;
```
Comment (qpfiffer) [inline templates/index.html]: @dequis :heart_eyes:

File: templates/index.html [hunk 2]
```diff
@@ -181,7 +184,16 @@
...
-            document.documentElement.style.background = '#' + this.value;
+            document.forms.new_bump['bg-color'].style.border = '4px solid #' + this.value;
+            document.forms.new_bump['bg-color'].style.boxShadow = '0 0 25px 5px #' + this.value;
+            document.forms.new_bump['bg-color'].style.background = '#' + this.value;
+            document.forms.new_bump['bg-color'].style.background;
+            document.forms.new_bump['bg-color'].style;
+            document.forms.new_bump['bg-color'];
+            document.forms.new_bump;
+            document.forms;
+            document;
```
Comment (qpfiffer) [inline templates/index.html]: the fuck is this

**Comment by qpfiffer** [comment]
Looks like it passes tests! Nice!

**Comment by Xe** [comment]
wait, you have tests?

**Comment by qpfiffer** [comment]
Nope! :japanese_ogre:

**Pull request merged**